src/arrayReverse.test.js

```diff
@@ -8,13 +8,42 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(arrayReverse([])).toEqual([]);
+  });
 
+  it(`should return an empty string`
+    + `, if original array consists of an empty string`, () => {
+    expect(arrayReverse(['', '', ''])).toEqual(['', '', '']);
   });
 
-  it(`should return an empty string
-    if original array consists of an empty string`, () => {
+  it(`should correctly return a reversed array`, () => {
     expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
   });
 
   // write more tests here
+  it(`should work properly`
+    + ` with strings with numbers and special symbols`, () => {
+    expect(
+      arrayReverse(['M@te', 'Ac@demy', '!s', '@', 't2ach1ng', 'pl0384719ace'])
+    )
+      .toEqual(['eca9', '174830l', 'pg', 'n', '1hca2t@s', '!ymed@cAet@M']);
```

Comment:
The expected output in this test case seems incorrect. The reversed array should maintain the original string lengths, but the expected output does not match this requirement. Please verify the expected result.